### ISSUE TITLE
Test reports in Java Test Runner are "Not Found"

### ISSUE THREAD

**Issue by emil-simeonov-se** [cdr/code-server#1090]
Not dublicates found.

- `code-server` version:
Docker image: codercom/code-server:v2
```bash
coder@5ad8c7a6c362:~/project$ code-server --version
info  2.1616-vsc1.39.2
info  cc7585bbc2aa35bad0074fa9f579a19e54132dc6
info  x64
```
**NB**
At least three versions of the v2 image were tested out. None of them worked. This is a dump of the most up-to-date one.

- OS Version: macOS Catalina 10.15, Docker Desktop: 2.1.0.4, web browsers: both up-to-date Firefox and Chrome
- Extension: https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-test

## Description

Whenever a Junit 4 or 5 test (as part of a simple Maven project) fails when run through Java Test Runner, a test report is supposed to be presented, so failures could be further investigated.

Instead an empty view (tabbed editor) is displayed with a simple error message in the upper left corner. It states: "Not Found". 

## Steps to Reproduce

1. Build the following Docker image:
```
FROM codercom/code-server:v2

ENV SHELL /bin/bash

RUN sudo apt update && \
    sudo apt install openjdk-11-jdk maven -y

ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64

# Install Java support extension
RUN code-server --install-extension vscjava.vscode-java-pack
```
2. Run a Docker container using the newly built image. Follow the instructions in [README.md](https://github.com/cdr/code-server/blob/master/README.md).
3. Create a simple Java Maven project with a dependency to JUnit 4 (test scope).
4. Write a test that fails (e.g. assertTrue(false)).
5. Execute the test.
6. Instead of a test report elaborating on the failure, you'll face an open side tabbed editor/viewer with an error message "Not Found".

**Additional Notes**
1. Java Test Runner extension works as expected on desktop:
```
Version: 1.39.2
Commit: 6ab598523be7a800d7f3eb4d92d7ab9a66069390
Date: 2019-10-15T15:33:00.827Z
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Darwin x64 19.0.0
```
2. While I'd love to see this issue solved, I could be unblocked by proposals for reasonable ways to troubleshoot failing JUnit tests. Running `mvn test` and going through the surefire test reports is not an option in my context.

**Comment by sr229** [comment]
Related to #1081

**Comment by emil-simeonov-se** [comment]
Well, they actually work for me. I can give hints about that in https://github.com/cdr/code-server/issues/1081, but it boils down to the default shell of the `code-server`.

The key to run and debug Java processes is to set bash as the default shell. This part in the Dockerfile provided by me does the magic.

`ENV SHELL /bin/bash`

**Comment by emil-simeonov-se** [comment]
I noticed that with `v2` many views (probably originating from user-installed extensions) behave in a similar way, showing "Not Found" only. But the same goes for other static assets like icons and folders... Could it be that this whole thing is caused by a more generic flaw related to finding, fetching, and displaying static resources (e.g. contributed by extensions)?

If that hypothesis seems seems likely, could anyone provide me with hints how to further troubleshoot this and get to the bottom of it? For example, pointing me to some files in the code-base, logs that could reveal more, etc. I could potentially get back with a PR.

**Comment by code-asher** [comment]
@emil-simeonov-se That's my fault; the latest `v2` image should be fixed.

Basically the view HTML was missing from the final build. https://github.com/cdr/code-server/commit/a6e4f967373068e5fc6b565b66c653610ff9b7e9

**Comment by code-asher** [comment]
Although I'm not sure about the icons. The commit I linked above only fixed the views (like extension readmes, test reports, and so on).

**Comment by code-asher** [comment]
I'll go ahead and close since the issue in the description should be resolved (if not let me know). For the icons it would be awesome if you could open a new issue describing the issues you're having with them.

**Comment by emil-simeonov-se** [comment]
@code-asher, I just ran `docker system prune -a` and re-built my own docker image. This way I verified I would be using the most up-to-date `v2` container. The fix works like charm!

I also do not observe issues with icons now. All the web views originating from extensions present their actual content.

Thank you so much for the ultra quick resolution! Thank you also for the amazing project and all the efforts you, guys, have been putting on it! Than you for sharing it with us.
